icons/pnpm/pnpm-plain.svg

```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M125.668 39.8047H86.6445V0.78125H125.668V39.8047Z" fill="#F9AD00"/>
+<path d="M82.7344 39.8047H43.707V0.78125H82.7344V39.8047Z" fill="#F9AD00"/>
+<path d="M39.8047 39.8047H0.78125V0.78125H39.8047V39.8047Z" fill="#F9AD00"/>
+<path d="M125.668 82.7305H86.6445V43.707H125.668V82.7305Z" fill="#F9AD00"/>
+<path d="M82.7344 82.7305H43.707V43.707H82.7344V82.7305Z" fill="#F9AD00"/>
+<path d="M82.7344 125.66H43.707V86.6328H82.7344V125.66Z" fill="#F9AD00"/>
+<path d="M125.668 125.66H86.6445V86.6328H125.668V125.66Z" fill="#F9AD00"/>
+<path d="M39.8047 125.66H0.78125V86.6328H39.8047V125.66Z" fill="#F9AD00"/>
+</svg>
```

Comment:
Looks good visually, but the paths need to be unified for plain and line icons.
See this guide for how to unite paths: https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473